### PR TITLE
ci: update chrome and firefox saucelabs versions

### PR DESCRIPTION
**Pull request by devversion**
* See individual commits.

### CODE REVIEW

File: browser-providers.conf.js [hunk 1]
```diff
@@ -43,10 +43,10 @@ var customLaunchers = {
   'DartiumWithWebPlatform':
       {base: 'Dartium', flags: ['--enable-experimental-web-platform-features']},
   'ChromeNoSandbox': {base: 'Chrome', flags: ['--no-sandbox']},
-  'SL_CHROME': {base: 'SauceLabs', browserName: 'chrome', version: '67'},
+  'SL_CHROME': {base: 'SauceLabs', browserName: 'chrome', version: '73'},
   'SL_CHROMEBETA': {base: 'SauceLabs', browserName: 'chrome', version: 'beta'},
   'SL_CHROMEDEV': {base: 'SauceLabs', browserName: 'chrome', version: 'dev'},
-  'SL_FIREFOX': {base: 'SauceLabs', browserName: 'firefox', version: '60'},
+  'SL_FIREFOX': {base: 'SauceLabs', browserName: 'firefox', version: '65'},
```
Comment (gkalpak) [inline browser-providers.conf.js]: Why not 66?

File: browser-providers.conf.js [hunk 2]
```diff
@@ -43,10 +43,10 @@ var customLaunchers = {
   'DartiumWithWebPlatform':
       {base: 'Dartium', flags: ['--enable-experimental-web-platform-features']},
   'ChromeNoSandbox': {base: 'Chrome', flags: ['--no-sandbox']},
-  'SL_CHROME': {base: 'SauceLabs', browserName: 'chrome', version: '67'},
+  'SL_CHROME': {base: 'SauceLabs', browserName: 'chrome', version: '73'},
```
Comment (gkalpak) [inline browser-providers.conf.js]: Given that we already test on Chrome `beta` and `dev` (which are not fixed versions), would it make sense to use [browser version shortcuts][1] (e.g. `latest`) for other browsers as well (so we don't have to manually update the versions)?

[1]: https://saucelabs.com/blog/new-browser-version-shortcuts

**Review by gkalpak — APPROVED** [review]

File: browser-providers.conf.js (hunk 1 above)
Comment (devversion) [reply browser-providers.conf.js]: Tried that initially, but it was not supported within Saucelabs yet. Can be also seen by [looking at their API](https://saucelabs.com/rest/v1/info/platforms/webdriver)

File: browser-providers.conf.js (hunk 2 above)
Comment (devversion) [reply browser-providers.conf.js]: We just saw what happened if we make us dependent on those browser shortcuts.

Ideally I'd like to lock the "beta" version as well, but I think the point of the beta testing is that it should give us awareness of potential failures ahead of time.

Comment (gkalpak) [reply browser-providers.conf.js]: We are already subject to non-determinism (due to ChromeBeta). Adding some more shouldn't have a noticeable impact (theoretically `latest` should not be less robust than `beta` :grin:) and we save having to manually update the version. Given that we already test with `beta`, I am not too concerned, so happy to keep things as they are if you think it switching to `latest` might increase instability.

Comment (devversion) [reply browser-providers.conf.js]: Fair enough 😁 I don't feel strong about it at all. Changed it to the version shortcuts!

Based on https://support.saucelabs.com/hc/en-us/articles/360017553593-Testing-on-Dev-and-Beta-Versions the stable channel should be tested thoroughly by the Saucelabs team as well.

File: packages/platform-browser/test/dom/shadow_dom_spec.ts [hunk 3]
```diff
@@ -27,9 +27,9 @@ if (browserDetection.supportsShadowDom) {
 
     it('should use the shadow root to encapsulate styles', () => {
       const compEl = TestBed.createComponent(StyledShadowComponent).nativeElement;
-      expect(window.getComputedStyle(compEl).border).toEqual('1px solid rgb(0, 0, 0)');
+      expect(window.getComputedStyle(compEl).backgroundColor).toEqual('rgb(0, 0, 0)');
```
Comment (kara) [inline packages/platform-browser/test/dom/shadow_dom_spec.ts]: Can we add a comment here about Firefox and why we can't use `border`? (similar to what's in the commit)

**Review by kara — CHANGES_REQUESTED** [review]
Looks good, but needs a comment

**Review by kara — APPROVED** [review]
LGTM

**Pull request closed without merging**